charts/SDFactory/values-cfx-int.yaml

```diff
@@ -0,0 +1,49 @@
+ingress:
```

Comment:
Same as above. Cofinity-X environment specifics do not belong in an open source project